### PR TITLE
added links in list response for paper check

### PR DESCRIPTION
**Pull request by madhankumar3103**
added links in list response for paper check

### CODE REVIEW

File: src/Hyperwallet/Response/ListResponse.php [hunk 1]
```diff
@@ -6,14 +6,28 @@
...
+     * Has Next Page objects
+     *
+     * @var boolean
+     */
+    private $hasNextPage;
+
+    /**
+     * Has Next Page objects
+     *
+     * @var boolean
```
Comment (arao6) [inline src/Hyperwallet/Response/ListResponse.php:28]: Pls update this comment - has Previous page

File: src/Hyperwallet/Response/ListResponse.php [hunk 2]
```diff
@@ -61,6 +84,33 @@ public function getData() {
...
+
+    /**
+     * Get Has Previous Page of Model's
+     *
+     * @var boolean
+     */
+    public function getHasPreviousPage() {
+        return $this->hasPreviousPage;
+    }
+
```
Comment (arao6) [inline src/Hyperwallet/Response/ListResponse.php:113]: offset and count fields/methods can be removed

Comment (madhankumar3103) [reply src/Hyperwallet/Response/ListResponse.php:113]: This Count and Offset method using array iteration propose , so we need those methods.

File: src/Hyperwallet/Response/ListResponse.php (hunk 1 above)
Comment (madhankumar3103) [reply src/Hyperwallet/Response/ListResponse.php:28]: This changes has been done , please check below commit id - 2c27e80a631550d33f0c3bddc01ecb067c5b008b

**Review by arao6 — APPROVED** [review]

**Review by Rrathinasabapath — APPROVED** [review]
Verified the changes.

**Review by vkorotchenko — APPROVED** [review]

**Pull request merged**